src/server/scripts/Northrend/Ulduar/HallsOfLightning/boss_volkhan.cpp

```diff
@@ -202,7 +202,7 @@ class boss_volkhan : public CreatureScript
                     summoned->GetMotionMaster()->MoveFollow(target, 0.0f, 0.0f);
 
                 // Why healing when just summoned?
-                summoned->CastSpell(summoned, SPELL_HEAT, false, nullptr, nullptr, me->GetGUID());
+                summoned->CastSpell(summoned, SPELL_HEAT, CastSpellExtraArgs().SetOriginalCaster(me->GetGUID()));
```

Comment:
just me->GetGUID?

Reply:
that would set triggered flag

Reply:
(in almost all cases, casts that pass caster GUID are triggered, same for aura/item - i'm fairly confident that the ones that _don't_ are bugs, but didn't want to change them in the refactor)